apps/papra-server/src/modules/documents/documents.routes.ts

```diff
@@ -422,3 +423,38 @@ function setupDeleteAllTrashDocumentsRoute({ app, config, db }: RouteDefinitionC
     },
   );
 }
+
+function setupUpdateDocumentRoute({ app, db }: RouteDefinitionContext) {
+  app.patch(
+    '/api/organizations/:organizationId/documents/:documentId',
+    validateParams(z.object({
+      organizationId: z.string().regex(organizationIdRegex),
+      documentId: z.string(),
```

Comment:
You may want to rebase with latest `main`, in #241 I recently mutualized the validation schemas

```suggestion
      organizationId: organizationIdSchema,
      documentId: documentIdSchema,
```